client/src/components/Footer/style.css

```diff
@@ -0,0 +1,25 @@
+.footer {
+    display: flex;
+    flex-direction: row;
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100vw;
+    justify-content: space-around;
+    background: #F7F7F7;
+}
+
+.footer_element img {
```

Comment:
you're not using BEM like this `.footer_element` this is the right way to write it `.footer__footer-element`